modules/dedicated-cluster-install-deploy.adoc

```diff
@@ -19,19 +19,27 @@ Cluster Logging Operator and Elasticsearch Operator.
 
 .Prerequisites
 
-Ensure that you have the necessary persistent storage for Elasticsearch. Note that each Elasticsearch node
-requires its own storage volume.
+You are provided with a maximum of 600GiB of persistent storage exclusively
+available for cluster logging related Elasticsearch deployments. Note that each
+Elasticsearch node requires its own storage volume.
 
-Elasticsearch is a memory-intensive application. Each Elasticsearch node needs 16G of memory for both memory requests and limits.
-The initial set of {product-title} nodes might not be large enough to support the Elasticsearch cluster. You must add additional nodes to the
-{product-title} cluster to run with the recommended or higher memory. Each Elasticsearch node can operate with a lower
-memory setting though this is not recommended for production deployments.
+Elasticsearch is a memory-intensive application. Each Elasticsearch node needs
```

Comment:
maybe
```
* Allocate enough memory for each Elasticsearch node to manage both memory requests and limits. For production environments, allocate 8 GiB of memory. You can allocate as little as <state_minimum_value> for non-production environments.
```

Will a user know how to do this? How low can you go with the memory?

Reply:
The 8gb is determined by the YAML, which is not really recommended to alter. I haven't heard of a lower setting, actually. Good point.

Reply:
That's good to know!